src/main.cpp

```diff
@@ -4,5 +4,6 @@
 int main()
 {
     std::cout << "LogicalImages!" << std::endl;
+
```

Comment:
is this necessary? Please revert this change.

Reply:
removed